test/Consul.Tests/App.config

```diff
@@ -4,11 +4,5 @@
     <ThrowUnobservedTaskExceptions enabled="false" />
     <gcServer enabled="true" />
     <gcConcurrent enabled="false" />
-    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
-      <dependentAssembly>
-        <assemblyIdentity name="System.Diagnostics.DiagnosticSource" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-4.0.1.0" newVersion="4.0.1.0" />
-      </dependentAssembly>
-    </assemblyBinding>
```

Comment:
@ReubenBond this was an issue in that particular PR, where I moved everything to auto-generated, but this one was still an issue